fuzzyfinder.go

```diff
@@ -263,6 +265,53 @@ func (f *finder) _draw() {
 	}
 }
 
+func parseColor(rs *[]rune) (tcell.Color, bool, error) {
```

Comment:
Error is never returned 👀 

```suggestion
func parseColor(rs *[]rune) (tcell.Color, bool) {
```